components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/codegen/CodeGenerator.java

```diff
@@ -74,18 +72,20 @@ public void generate(String projectName, boolean overwrite)
         String projectSrcPath = CmdUtils.getProjectTargetModulePath((projectName));
         List<GenSrcFile> genFiles = new ArrayList<>();
         List<BallerinaService> serviceList = new ArrayList<>();
-        List<BallerinaService> openAPIServiceList = new ArrayList<>();
-        List<String> openAPIDirectoryLocations = new ArrayList<>();
-        String projectAPIDefGenLocation = CmdUtils.getProjectGenAPIDefinitionPath(projectName);
-        openAPIDirectoryLocations.add(CmdUtils.getProjectDirectoryPath(projectName) + File.separator
-                + CliConstants.PROJECT_API_DEFINITIONS_DIR);
         String grpcDirLocation = CmdUtils.getGrpcDefinitionsDirPath(projectName);
-        if (Files.exists(Paths.get(projectAPIDefGenLocation))) {
-            openAPIDirectoryLocations.add(projectAPIDefGenLocation);
-        }
         CodeGenerator.projectName = projectName;
         BallerinaToml ballerinaToml = new BallerinaToml();
         ballerinaToml.setToolkitHome(CmdUtils.getCLIHome());
+        List<String> openAPIDirectoryLocations = new ArrayList<>();
```

Comment:
What is the usage of this list. Only paths are added to it., But never read from it

Reply:
Used to iterate over all openApi paths within the project. Usage is at line 92